packages/core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe core SDK package will be documented in this file.
 
+## Recent Changes
+
+- Enhancement: Added recursive optional header to `ZosmfHeaders`. [#2436](https://github.com/zowe/zowe-cli/pull/2436)
```

Comment:
- Is there a word missing here?: "Added recursive optional header ... "
- We should explain what `ZosmfHeaders` is: " ... to the **file/profile/whatever** `ZosmfHeaders`."